administration/src/application/components/forms/StepSendForm.tsx

```diff
@@ -46,23 +50,45 @@ const StepSendForm: Form<StepSendFormState, Options, ValidatedInput, AdditionalP
       },
     }
   },
-  Component: ({ state, setState }) => (
-    <>
-      <CheckboxForm.Component
-        label='Ich erkläre mich damit einverstanden, dass meine Daten zum Zwecke der Antragsverarbeitung
-              gespeichert werden.'
-        state={state.acceptedDataPrivacy}
-        setState={useUpdateStateCallback(setState, 'acceptedDataPrivacy')}
-        options={acceptedDatePrivacyOptions}
-      />
-      <CheckboxForm.Component
-        label='Ich versichere, dass alle angegebenen Informationen korrekt und vollständig sind.'
-        state={state.givenInformationIsCorrectAndComplete}
-        setState={useUpdateStateCallback(setState, 'givenInformationIsCorrectAndComplete')}
-        options={givenInformationIsCorrectAndCompleteOptions}
-      />
-    </>
-  ),
+  Component: ({ state, setState, privacyPolicy }) => {
+    const [openPrivacyPolicy, setOpenPrivacyPolicy] = useState<boolean>(false)
+    const PrivacyLabel = (
+      <div style={{ alignSelf: 'center' }}>
+        Ich erkläre mich damit einverstanden, dass meine Daten zum Zwecke der Antragsverarbeitung gespeichert werden und
+        akzeptiere die{' '}
+        <Button
+          variant='text'
+          style={{ textTransform: 'capitalize', padding: 0, verticalAlign: 'unset' }}
+          onClick={() => setOpenPrivacyPolicy(true)}>
+          Datenschutzerklärung
+        </Button>
+      </div>
+    )
+    return (
+      <>
+        <CheckboxForm.Component
+          state={state.acceptedDataPrivacy}
+          setState={useUpdateStateCallback(setState, 'acceptedDataPrivacy')}
+          options={acceptedDatePrivacyOptions}
+          label={PrivacyLabel}
+        />
+
+        <CheckboxForm.Component
+          label='Ich versichere, dass alle angegebenen Informationen korrekt und vollständig sind.'
+          state={state.givenInformationIsCorrectAndComplete}
+          setState={useUpdateStateCallback(setState, 'givenInformationIsCorrectAndComplete')}
+          options={givenInformationIsCorrectAndCompleteOptions}
+        />
+        <BasicDialog
+          open={openPrivacyPolicy}
+          maxWidth='lg'
+          onUpdateOpen={() => setOpenPrivacyPolicy(false)}
```

Comment:
Either make this `onClose` or use 
```suggestion
          onUpdateOpen={setOpenPrivacyPolicy}
```